Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml

```diff
@@ -32,6 +32,9 @@
     tags:
     - CaptainSabre
   - type: DisarmMalus
+  - type: Utensil
```

Comment:
just make BaseSword that has this and any other copy pasted components

Reply:
sounds like a more difficult path, but.. if that actually makes sense..